src/main/java/org/jenkinsci/plugins/casc/integrations/GroupPermissionDefinition.java

```diff
@@ -0,0 +1,47 @@
+package org.jenkinsci.plugins.casc.integrations;
+
+import hudson.security.Permission;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import java.util.*;
+
+/**
+ * Created by mads on 2/7/18.
+ */
+public class GroupPermissionDefinition {
```

Comment:
Restrict classes so that they do not become a part of public API

Reply:
done